exampleCourse/questions/demo/autograder/c/square/info.json

```diff
@@ -1,6 +1,6 @@
 {
     "uuid": "1e1a7e46-1511-49b8-8869-744276a309a0",
-    "title": "Autograder demo: Square function in C",
+    "title": "Autograder demo: Square function in C (Standard Output Override)",
```

Comment:
What does "override" imply in the name?

Reply:
The main function in this code replaces the standard output to ensure the provided code does not print some result that could be acceptable in the autograder. In other words, it ensures that students return the value, not print it.

Reply:
👍 thanks for explaining!